src/components/Posts/ShowcaseArt.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <q-carousel
+    animated
+    arrows
+    control-color="primary"
+    height="auto"
```

Comment:
```height = <absolute value>```
May work here

Reply:
Not sure, height may vary due to Artist slide.
I'm going to apply a fit `cover` to the image instead of a `contain`.